libs/community/langchain_community/chat_models/litellm.py

```diff
@@ -132,10 +136,30 @@ def _convert_delta_to_message_chunk(
     else:
         additional_kwargs = {}
 
+    tool_call_chunks = []
+    if raw_tool_calls := _dict.get("tool_calls"):
+        additional_kwargs["tool_calls"] = raw_tool_calls
```

Comment:
`raw_tool_calls` should probably be a dict or something readily json serializable. I think we may need to call `.model_dump()` on the tool calls.